profiles/seccomp/seccomp.go

```diff
@@ -110,6 +126,13 @@ Loop:
 				}
 			}
 		}
+		if call.Excludes.MinKernel != "" {
+			if ok, err := kernelGreaterEqualThan(call.Excludes.MinKernel); err != nil {
+				return nil, err
```

Comment:
in the error case should we not just include the rule anyway? ie fail closed not error.

Reply:
This should fail only in the case where profile had a version that couldn't be parsed. I think that case is similar to the other validation, eg. the `call.Names` check.